Makefile

```diff
@@ -5,8 +5,8 @@ TOOLS_BIN_DIR := $(abspath ./build/tools)
 LINTER = $(TOOLS_BIN_DIR)/golangci-lint
 
 WIN_BUILD = GOOS=windows GOARCH=amd64 go build -trimpath -o $(BUILD_SPACE)/validator/windows/amd64
-LINUX_AMD64_BUILD = GOOS=linux GOARCH=amd64 go build -trimpath -o $(BUILD_SPACE)/validator/linux/amd64
-LINUX_ARM64_BUILD = GOOS=linux GOARCH=arm64 go build -trimpath -o $(BUILD_SPACE)/validator/linux/arm64
+LINUX_AMD64_BUILD = CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -trimpath -o $(BUILD_SPACE)/validator/linux/amd64
+LINUX_ARM64_BUILD = CGO_ENABLED=0  GOOS=linux GOARCH=arm64 go build -trimpath -o $(BUILD_SPACE)/validator/linux/arm64
```

Comment:
Why is it necessary to disable CGO here?
There are some benefits to using the default (enabled):
https://stackoverflow.com/questions/64531437/why-is-cgo-enabled-1-default

Reply:
The problem with that would be: if turning on `CGO_ENABLED=1` and building in Debian and doing a cross compilation and putting up into an AL2, then the AL2 host require load the host OS's native libraries (in this case, its `glibc`). However, different OS has different GLIB version and Github Action; therefore, causing the different in cross compilation with the following error 
```
null_resource.performance (remote-exec): ./validator: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by ./validator)
null_resource.performance (remote-exec): ./validator: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by ./validator)
```

**Note**:  AL2 by default having a [glibc of 2.26 while Al2022 has GLIBC 2.32](https://repost.aws/questions/QUrXOioL46RcCnFGyELJWKLw/glibc-2-27-on-amazon-linux-2). So it would be better to have a build that can cross compile all OS instead of OS that has specific GLIBC version.

Reply:
There is no plan or reason to compile the test code on darwin and copy it to an AL2 host and run it there.

Doesn't the test code get copied to the host-under-test and run directly on it?

Reply:
The validator is not a `test package` but an [validator application](https://github.com/aws/amazon-cloudwatch-agent-test/tree/main/validator). So yes it is in need to compile the test code in X host and put it on Y host. Windows has problems with allocation memory when using `go test` so its only possible to use validator